### ISSUE TITLE
Update to E.F.A. 3.0.0.9 failed

### ISSUE THREAD

**Issue by Bepsi** [E-F-A/v3#330]
Hi,

When I tried to update my EFA server to 3.0.0.9 version, I have this message:

```
---> Package postfix-perl-scripts.x86_64 2:2.6.6-6.el6_7.1 will be an update
--> Processing Dependency: postfix = 2:2.6.6-6.el6_7.1 for package: 2:postfix-perl-scripts-2.6.6-6.el6_7.1.x86_64
--> Finished Dependency Resolution
Error: Package: 2:postfix-perl-scripts-2.6.6-6.el6_7.1.x86_64 (base)
           Requires: postfix = 2:2.6.6-6.el6_7.1
           Removing: 2:postfix-2.6.6-6.el6_5.x86_64 (@updates)
               postfix = 2:2.6.6-6.el6_5
           Updated By: 2:postfix-3.0.4-1.efa.el6.x86_64 (EFA)
               postfix = 2:3.0.4-1.efa.el6
           Available: 2:postfix-2.6.6-6.el6_7.1.x86_64 (base)
               postfix = 2:2.6.6-6.el6_7.1
           Available: 2:postfix-3.0.3-2.efa.el6.x86_64 (EFA)
               postfix = 2:3.0.3-2.efa.el6
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
Update to Version 3.0.0.9 FAILED.  Updates cancelled.
Please visit http://www.efa-project.org for more information.
```

YUM give me the same message:
```
Loaded plugins: fastestmirror, security
Setting up Install Process
Loading mirror speeds from cached hostfile
 * EFA: dl.efa-project.org
 * base: centos.mirrors.ovh.net
 * epel: mirrors.ircam.fr
 * extras: centos.weezel.co.uk
 * updates: centos.trisect.eu
Resolving Dependencies
--> Running transaction check
---> Package postfix.x86_64 2:2.6.6-6.el6_7.1 will be updated
--> Processing Dependency: postfix = 2:2.6.6-6.el6_7.1 for package: 2:postfix-perl-scripts-2.6.6-6.el6_7.1.x86_64
---> Package postfix.x86_64 2:3.0.4-1.efa.el6 will be an update
--> Finished Dependency Resolution
Error: Package: 2:postfix-perl-scripts-2.6.6-6.el6_7.1.x86_64 (@base)
           Requires: postfix = 2:2.6.6-6.el6_7.1
           Removing: 2:postfix-2.6.6-6.el6_7.1.x86_64 (@base)
               postfix = 2:2.6.6-6.el6_7.1
           Updated By: 2:postfix-3.0.4-1.efa.el6.x86_64 (EFA)
               postfix = 2:3.0.4-1.efa.el6
           Available: 2:postfix-3.0.3-2.efa.el6.x86_64 (EFA)
               postfix = 2:3.0.3-2.efa.el6
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

Can you help me ?

Thank you.

**Comment by shawniverson** [comment]
```
sudo rpm -e postfix-perl-scripts

**Comment by Bepsi** [comment]
Hi,

Same error:

```
# rpm -qa | grep postfix
postfix-3.0.4-1.efa.el6.x86_64
```

```
# yum install postfix-perl-scripts
Loaded plugins: fastestmirror, security
Setting up Install Process
Loading mirror speeds from cached hostfile
 * EFA: dl.efa-project.org
 * base: centos.trisect.eu
 * epel: epel.mirrors.ovh.net
 * extras: fr.mirror.babylon.network
 * updates: mirrors.ircam.fr
Resolving Dependencies
--> Running transaction check
---> Package postfix-perl-scripts.x86_64 2:2.6.6-6.el6_7.1 will be installed
--> Processing Dependency: postfix = 2:2.6.6-6.el6_7.1 for package: 2:postfix-perl-scripts-2.6.6-6.el6_7.1.x86_64
--> Finished Dependency Resolution
Error: Package: 2:postfix-perl-scripts-2.6.6-6.el6_7.1.x86_64 (base)
           Requires: postfix = 2:2.6.6-6.el6_7.1
           Installed: 2:postfix-3.0.4-1.efa.el6.x86_64 (@EFA)
               postfix = 2:3.0.4-1.efa.el6
           Available: 2:postfix-2.6.6-6.el6_7.1.x86_64 (base)
               postfix = 2:2.6.6-6.el6_7.1
           Available: 2:postfix-3.0.3-2.efa.el6.x86_64 (EFA)
               postfix = 2:3.0.3-2.efa.el6
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

**Comment by Bepsi** [comment]
Hi,

I re-run /usr/local/sbin/EFA-Update ... and all is OK.

Thank you.
